### PR TITLE
Switch to contribution token instead of proposals

### LINKED ISSUE #20: Load contributions from Contribution contract/storage
<del>As we'll lose the proposals themselves every time we update the Kredits contract, we need to show recent/past contributions from the Token contract data instead.

@bumi I'm not quite sure how we can implement this. Could you provide a rough outline of we'd have to change?</del>

Update: we're in the process of implementing ERC721 tokens for contributions, so all data will move to a Contribution contract/storage. See https://github.com/67P/kredits-contracts/issues/52

### PR DESCRIPTION
**Pull request by skddc**
No vetos yet, and only for collections (not creation).

Also removes Bourbon and Neat in favor of plain Sass.

closes #20

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -12,26 +14,44 @@ export default Controller.extend({
...
-  proposalsClosed: filterBy('proposals', 'isExecuted', true),
-  proposalsSorting: Object.freeze(['id:desc']),
-  proposalsClosedSorted: sort('proposalsClosed', 'proposalsSorting'),
-  proposalsOpenSorted: sort('proposalsOpen', 'proposalsSorting'),
+  contributions: alias('kredits.contributions'),
+  contributionsSorting: Object.freeze(['id:desc']),
+
+  contributionsUnconfirmed: computed('contributions.[]', 'currentBlock', function() {
+    return this.contributions.filter(contribution => {
+      return contribution.confirmedAt < this.currentBlock;
```
Comment (bumi) [inline app/controllers/index.js]: I think this should be the other way round. 
unconfirmed = confirmedAt > currentBlock

or: 
if the currentBlock is smaller than the block number in confirmedAt then the contribution is not confirmed.

File: app/controllers/index.js [hunk 2]
```diff
@@ -12,26 +14,44 @@ export default Controller.extend({
...
+    });
+  }),
+  contributionsConfirmed: computed('contributions.[]', 'currentBlock', function() {
+    return this.contributions.filter(contribution => {
+      return contribution.confirmedAt >= this.currentBlock;
```
Comment (bumi) [inline app/controllers/index.js]:
```suggestion
      return this.currentBlock >= contribution.confirmedAt;
```

File: app/controllers/index.js (hunk 1 above)
Comment (bumi) [reply app/controllers/index.js]:
```suggestion
      return this.currentBlock < contribution.confirmedAt;
```

File: app/services/kredits.js [hunk 3]
```diff
@@ -83,21 +87,23 @@ export default Service.extend({
...
         addresses: { Kernel: config.kreditsKernelAddress },
         apm: config.kreditsApmDomain,
         ipfsConfig: config.ipfs
       });
       return kredits
         .init()
-        .then((kredits) => {
+        .then(async (kredits) => {
           this.set('kredits', kredits);
+          this.set('currentBlock', await kredits.provider.getBlockNumber());
```
Comment (bumi) [inline app/services/kredits.js]: do we actually need to await this here? 
currentBlock is not used in this function it seems?

Comment (skddc) [reply app/services/kredits.js]: We don't want to set it to a promise I think.

File: app/controllers/index.js (hunk 1 above)
Comment (skddc) [reply app/controllers/index.js]: Aw crap, of course. Brain fart.

File: package.json [hunk 4]
```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
-    "start": "ember serve",
     "test": "ember test",
+    "start": "KREDITS_APM_DOMAIN=open.aragonpm.eth ember serve",
```
Comment (bumi) [inline package.json]: :+1:

File: app/services/kredits.js (hunk 3 above)
Comment (bumi) [reply app/services/kredits.js]: ok, I thought we want to set a promise that ember automatically resolves... like `getContributions` etc. return promises.

**Comment by bumi** [comment]
we soon also need to load the balances from the confirmed contributions or add more information on what we show. otherwise it is a bit misleading.

File: app/services/kredits.js (hunk 3 above)
Comment (skddc) [reply app/services/kredits.js]: That's different. `getContributions` doesn't set a property directly. Wherever it's used, the promise is actually handled.

**Comment by skddc** [comment]
> we soon also need to load the balances from the confirmed contributions or add more information on what we show. otherwise it is a bit misleading.

Yes, but we currently don't get the correct balance of kredits earned via `getContributors`. It's a new issue, but I agree that it is the next urgent one.

File: app/services/kredits.js (hunk 3 above)
Comment (skddc) [reply app/services/kredits.js]: (That is to say the model hooks are promise-aware)

**Pull request merged**